src/javascripts/templates/budgetTemplate.js

```diff
@@ -0,0 +1,56 @@
+import { formatDate, formatNumber } from '../helpers/format';
+
+class BudgetTemplate {
+    /**
+     * Constructor of TemplateBudget object
+     */
+    constructor() {}
+
+    /**
+     * HTML TemplateBudget for render an Budget object
+     * @param {object} budget
+     */
+    static createBudget = (budget) => {
+        const budgetDateTransform = formatDate(budget.date);
+        const budgetDate = budgetDateTransform.split(' ');
+
+        return `
+          <li class="transaction-detail">
+            <div class="transaction-info">
+              <div class="datetime">
+                <p class="date-text">${new Date(budget.date).getDate()}</p>
+                <div>
```

Comment:
WHat is this div use for?

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!